wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/wdlom2wom/ValueGeneratorAndConsumer.scala

```diff
@@ -0,0 +1,22 @@
+package wdl.draft3.transforms.wdlom2wom
+
+import simulacrum.typeclass
+import scala.language.implicitConversions
+
+@typeclass
+trait ValueGenerator[A] {
```

Comment:
ToL Any way to be more specific?   By name it looks like it could create any value but by type sig looks like it only produces values of type string, seems like there's a potential for clarification.

Reply:
Ah, yeah, it's actually finding the *names* of values that this `A` will generate. Maybe I could rename the def to `def generatedValueNames`.